app/index.js

```diff
@@ -1,5 +1,4 @@
-import React, { View, Text
-} from 'react-native';
```

Comment:
**<a href='http://eslint.org/docs/rules/no-unused-vars'>no-unused-vars</a>**: "ScrollView" is defined but never used